### PR TITLE
Add pyMannKendall from PyPI

### PR DESCRIPTION
**Pull request by dpryan79**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml" 
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by dpryan79** [comment]
Needed by https://github.com/bioconda/bioconda-recipes/pull/19849

File: recipes/pymannkendall/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a10a1cc24c7466bce7e1c0c70c5601ad44aee573d2ce161e6e68b60f6a94e11e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+
```
Comment (npavlovikj) [inline recipes/pymannkendall/meta.yaml]: This can be `noarch`, right?

File: recipes/pymannkendall/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+requirements:
+  host:
+    - numpy
```
Comment (npavlovikj) [inline recipes/pymannkendall/meta.yaml]: Usually only `python` and `pip` should be in the `host` section. Unless they are really needed for building, I would suggest removing `numpy` and `scipy` from here.

File: recipes/pymannkendall/meta.yaml (hunk 1 above)
Comment (dpryan79) [reply recipes/pymannkendall/meta.yaml]: Yes, I always forget that the skeleton doesn't do that.

File: recipes/pymannkendall/meta.yaml (hunk 2 above)
Comment (dpryan79) [reply recipes/pymannkendall/meta.yaml]: My understanding was that conda-forge wanted run-time requirements in the host section (as opposed to bioconda, where they're removed).

Comment (npavlovikj) [reply recipes/pymannkendall/meta.yaml]: I am not aware of that, and based on few reviews I have seen and https://conda-forge.readthedocs.io/en/latest/meta.html#use-pip (the last paragraph), I thought the opposite was the case. However, I am ok with merging the recipe either way.

Comment (dpryan79) [reply recipes/pymannkendall/meta.yaml]: Let's just merge it then, it won't have any real tangible effect which way we handle the `host` section :)

**Pull request merged**